src/libs/actions/Report.js

```diff
@@ -1084,6 +1086,38 @@ function deleteReportComment(reportID, reportAction) {
                 };
 
                 Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`, reportActionsToMerge);
+            } else {
+                // Update last message information in the report object
+                Onyx.connect({
+                    key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`,
+                    callback: _.once((reportActionList) => {
+                        // Find last non-empty message
+                        const lastReportAction = !_.isEmpty(reportActionList)
+                            ? _.find(Object.values(reportActionList).reverse(),
+                                ({message, actionName}) => _.last(message).html !== ''
+                                && actionName !== 'CREATED')
+                            : null;
+
+                        // Populate last message details
+                        const lastMessageDetails = {};
+                        if (lastReportAction) {
+                            // Replace line breaks with spaces and remove all HTML from the message
+                            lastMessageDetails.lastMessageText = lastReportAction.isAttachment
+                                ? '[Attachment]'
+                                : _.last(lastReportAction.message)
+                                    .html.replace(/((<br[^>]*>)+)/gi, ' ')
+                                    .replace(/(<([^>]+)>)/gi, '');
```

Comment:
I think it could be nice to use `Str.stripHTML` like we do [here](https://github.com/Expensify/Expensify.cash/blob/main/src/libs/actions/Report.js#L1240), I see your regex is slightly different but I think the `Str.stripHTML` regex works just as well, what do you think?

Reply:
Actually that regex is not mine it's a copy pasta from here:

https://github.com/Expensify/Expensify.cash/blob/main/src/libs/actions/Report.js#L170-L172

Your suggestion really only removes one regex since we still need one that turns `<br>` into spaces, but since it's barely readable anyway, I see no harm in replacing it.

I'll test the change later and let you know.

Reply:
Aah nice, well I believe they're for the same purpose, and I think eventually we'll end up replacing them all with `Str.stripHTML` 🤷  thanks in advance for testing it out 👍

Reply:
@Beamanator  Replaced one of replaces with the `Str.stripHTML` call. Seems to work as expected.